testing/stress-ng/APKBUILD

```diff
@@ -0,0 +1,60 @@
+# Contributor: Olliver Schinagl <oliver@schinagl.nl>
+# Maintainer: Olliver Schinagl <oliver@schinagl.nl>
+pkgname=stress-ng
+pkgver=0.09.54
+pkgrel=0
+pkgdesc="stress-ng will stress test a computer system in various selectable ways."
+url="https://kernel.ubuntu.com/~cking/stress-ng/"
+arch="all"
+license="GPL-2.0-or-later"
+depends="
+	libc6-compat
+	libgcrypt
+"
+makedepends="
+	attr-dev
+	keyutils-dev
+	libaio-dev
+	libbsd-dev
+	libcap-dev
+	libgcrypt-dev
+	linux-headers
+	lksctp-tools-dev
+	zlib-dev
+"
+options="!check" # Tests are unreliable with qemu and take a long time
+subpackages="${pkgname}-doc ${pkgname}-examples"
+source="
+	https://kernel.ubuntu.com/~cking/tarballs/${pkgname}/${pkgname}-0.09.54.tar.xz
+
```

Comment:
redundant blank line

Reply:
actually, it is the separator between the source and the patch. I saw a lot of packages making that distinction which, for readability, makes sense.